$NetBSD: patch-configure.ac,v 1.1 2020/10/01 20:14:56 nils Exp $

Fix *BSD compilation : secure_getenv is GNU-only, so unavailable on BSD OS.

--- configure.ac.orig	2020-06-07 14:39:56.000000000 +0000
+++ configure.ac
@@ -166,7 +166,7 @@ AC_SUBST(COMPRESS_EXT)
 AC_DEFINE_UNQUOTED([ROOT_UID], [0], [Root user-id.])
 AC_SUBST(ROOT_UID)
 
-AC_CHECK_FUNCS([asprintf madvise qsort strndup strptime utimensat vsyslog])
+AC_CHECK_FUNCS([asprintf madvise qsort secure_getenv strndup strptime utimensat vsyslog])
 AC_CONFIG_HEADERS([config.h])
 
 AM_CFLAGS="\
